roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -18,25 +18,25 @@
   tags: ngi_reports
 
 - name: Unpack TeXLive
-  unarchive:
-    src: "{{ texLive_dest }}/src/{{ texLive_file }}"
-    dest: "{{ texLive_dest }}/src/"
-    copy: no
-    creates: "{{ texLive_dest }}/src/install-tl-*/LICENSE.TL"
-    mode: g+rw
+  command: "tar -xzf {{ texLive_dest }}/src/{{ texLive_file }} -C {{ texLive_dest }}/src/ --strip-components=1"
```

Comment:
Are you trying to unpack everything directly under `{{ texLive_dest }}/src/`? Because on L37 you're launching `install-tl` directly from this dir.